tutorials/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -388,6 +388,53 @@ hex_encode                 StringExtensions.HexEncode (Consider using `System.Co
 .. _System.Uri.EscapeDataString: https://learn.microsoft.com/en-us/dotnet/api/system.uri.escapedatastring
 .. _System.Uri.UnescapeDataString: https://learn.microsoft.com/en-us/dotnet/api/system.uri.unescapedatastring
 
+Signal
+------
+
+The following methods were converted to properties with their respective names changed:
+
+====================  ==============================================================
+GDScript              C#
+====================  ==============================================================
+``get_name()``        ``Name``
+``get_object()``      ``Owner``
+====================  ==============================================================
+
+The ``Signal`` type implements the awaitable pattern which means it can be used with
+the ``await`` keyword. See :ref:`_doc_c_sharp_differences_await`.
+
+Instead of using the ``Signal`` type, the recommended way to use Godot signals in C# is
+to use the generated C# events. See :ref:`doc_c_sharp_signals`.
+
+Callable
+--------
+
+The following methods were converted to properties with their respective names changed:
+
+====================  ==============================================================
+GDScript              C#
+====================  ==============================================================
+``get_object()``      ``Target``
+``get_method()``      ``Method``
+====================  ==============================================================
+
+Currently C# doesn't support ``CustomCallable``s which means ``Callable``s coming from
```

Comment:
Is that so? I've been calling `Callable`s defined in GDScript without issue 🤔

Reply:
Have you tried a GDScript `Callable` with bound variables?

Reply:
Indeed, doesn't work with bound variables. But this line sounds like it doesn't work, whatever the case is. It's fine if we want to document it as generally not working or "unknown behaviour" to keep the doc clean, though. People who want to try it still can.

Reply:
Yeah, I thought about it for the past hour and to be honest I don't really have a better suggestion.

Reply:
How about this?

> Currently C# only supports ``Callable``s created in C# as well, ``Callable``s coming from other languages may not work.

Honestly, it doesn't seem much better to me.

Reply:
Yeah, potayto, potahto.

Reply:
I'm concerned about the uncertainty in either phrasing. Do we not know what would happen?

Reply:
> Do we not know what would happen?

As far as I understand, "basic" `Callable`s aren't an issue. Only once you start binding arguments to them will it not work.

Reply:
Then we should state exactly that, clearly, I believe.

Reply:
Yes, "basic" `Callable`s work. What doesn't work are `CallableCustom` implementations such as `CallableCustomBind` or `CallableCustomUnbind` (can be created in GDScript using the `bind` and `unbind` methods); or any other implementation such as one provided by a GDExtension (e.g.: a Rust `Callable`).

I wanted to avoid mentioning bounded `Callable`s specifically because those aren't the only type of `Callable` that doesn't work (any `CallableCustom` won't work other than `ManagedCallable`, which is the C# one).

But I've been struggling to think of a better way to express all of this in a concise manner and without scaring users about using `Callable` at all since "basic" ones still work. I'm open to suggestions.

Reply:
Then I think the problem with your current phrasing is that "CustomCallable" is never explained in the article, nor linked externally. I think that you should state these things:

1. C# supports its own native implementation of `Callable` called `ManagedCallable`.
2. C# supports the basic version of the engine's `Callable`, i.e. a method without any bound information.
3. C# does not support engine's `Callable`s with bound information, such as created through `bind`/`unbind`.
4. C# is unlikely to work with implementations of `Callable`s from other languages through the GDExtension API.